src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultActions = [];
+  const copiedObject = { ...state };
```

Comment:
It's not a good practice to use variable type in variable name

You can rename it to stateCopy or stateClone